### PR TITLE
Accept nurseconnect registrations

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: registrations/tasks.py [hunk 1]
```diff
@@ -119,6 +124,28 @@ def check_operator_id(self, data_fields, registration):
         else:
             return []
 
+    def check_msisdn_registrant(self, data_fields, registration):
+        if "msisdn_registrant" not in data_fields:
+            return ["Msisdn of Registrant missing"]
```
Comment (imsickofmaps) [inline registrations/tasks.py]: `Msisdn` should be `MSISDN`

File: registrations/tasks.py [hunk 2]
```diff
@@ -198,7 +231,7 @@ def create_subscriptionrequests(self, registration):
             "identity": registration.registrant_id,
             "messageset": msgset_id,
             "next_sequence_number": next_sequence_number,
-            "lang": registration.data["language"],
+            "lang": "eng_ZA",  # NurseConnect is currently only in english
```
Comment (imsickofmaps) [inline registrations/tasks.py]: Not sure this is something worth putting in code in here. The only reason for this is lack of translated content. I think we should move this constraint to the jsbox app only.

**Comment by imsickofmaps** [comment]
Minor changes then 👍  when travis happy

**Pull request merged**